cargo/deprecation_warning.go

```diff
@@ -1,33 +0,0 @@
-package cargo
-
-import (
-	"fmt"
-
-	"github.com/BurntSushi/toml"
-)
-
-type DeprecationWarning struct{}
-
-func NewDeprecationWarning() DeprecationWarning {
-	return DeprecationWarning{}
-}
-
-func (d DeprecationWarning) WarnDeprecatedFields(path string) error {
```

Comment:
Why do we need to get rid of deprecation warning code? Won't we need it in the future?

Reply:
This code was to warn that the `include_files` and `pre_package` fields are deprecated. Now that we've moved all the existing buildpacks over, I don't think we still need this. Do you see any concerns with that?

Reply:
I realize now that this code is hardcoded to look for those fields, so we don't need that anymore. However, I do think we will probably need to reintroduce code like this in the future for deprecations. It might make more sense to re-write this code at that time to be more generic. 

TLDR is I think I'm good getting rid of this now that I realize it's specific to certain deprecated fields.